charts/mothership/templates/env.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: env-cm
+  namespace: {{ .Release.Namespace }}
+data:
+  L1_RPC: {{ .Values.env.L1_RPC }}
+  L1_BEACON_RPC: "{{ .Values.env.L1_BEACON_RPC }}"
```

Comment:
if these are only used with env vars, we can inject directly, not through configmap like below
```
env:
  - name: L1_RPC
    value: "{{ .Values.env.L1_RPC }}"
```
or is there another reason we should use configmap?

Reply:
yup there's no reason to keep that configmap by now. it was legacy from manual pre-helm yamls xP